src/httpserver.cpp

```diff
@@ -93,9 +93,10 @@ class WorkQueue
 
 public:
     WorkQueue(size_t _maxDepth) : running(true),
-                                 maxDepth(_maxDepth),
-                                 numThreads(0)
+                                 maxDepth(_maxDepth)
     {
+        std::lock_guard<std::mutex> lock(cs);
```

Comment:
This is really confusing. I don't think we should be making this nonsensical change just to make a sanitizer tool happy.